### PR TITLE
Mes-334 all on one common - improve undo button

### PR DESCRIPTION
**Pull request by zielinskikamil**
Improved undo button
screen shot 2018-03-13 at 14 52 15

### CODE REVIEW

File: src/components/totals/totals.html [hunk 1]
```diff
@@ -3,5 +3,7 @@
   <span class="report-header-text scorecard-item">D/F: {{faultMarks$ | async}}</span>
   <span class="report-header-text scorecard-item">S: {{seriousMarks$ | async}}</span>
   <span class="report-header-text scorecard-item">D: {{dangerousMarks$ | async}}</span>
-  <ion-icon (click)="undoMark()" name="undo"></ion-icon>
+  <button class="report-header-text scorecard-item">
+    <ion-icon (click)="undoMark()" name="undo"></ion-icon>
```
Comment (danielchildsbjss) [inline src/components/totals/totals.html]: should (click) be on the button?

File: src/components/totals/totals.html [hunk 2]
```diff
@@ -3,5 +3,7 @@
   <span class="report-header-text scorecard-item">D/F: {{faultMarks$ | async}}</span>
   <span class="report-header-text scorecard-item">S: {{seriousMarks$ | async}}</span>
   <span class="report-header-text scorecard-item">D: {{dangerousMarks$ | async}}</span>
-  <ion-icon (click)="undoMark()" name="undo"></ion-icon>
+  <button class="report-header-text scorecard-item">
```
Comment (ammarhaiderbjss) [inline src/components/totals/totals.html]: Can we make this button different from the other scorecard items? Perhaps a filled button like 'End'? What do you guys think?

Comment (zielinskikamil) [reply src/components/totals/totals.html]: hym, it would be to busy if we do another big button next to End button?

Comment (ammarhaiderbjss) [reply src/components/totals/totals.html]: I was just thinking colour, size can stay the same.

**Pull request merged**